f.el

```diff
@@ -270,6 +270,23 @@ If APPEND is non-nil, append the DATA to the existing contents."
         (unless (f-directory? path)
           (f--destructive path (make-directory path)))))))
 
+(defun f-mkdir-recursive (&rest dirs)
+  "Recursively create each directory in DIRS."
+
+  (dolist (dir dirs)
+    (let (dirs-to-mkdir)
+
+      (f-traverse-upwards
+       (lambda (path)
+	 (let ((f-exists-p (f-exists? path)))
```

Comment:
There are also a couple of tabs in your code, could you replace them with spaces?